cmd/foxtrot/main.go

```diff
@@ -1,27 +1,71 @@
 package main
 
 import (
+	"bufio"
 	"log"
+	"net"
 	"net/http"
-	"os"
 
+	"foxygo.at/foxtrot/pkg/foxtrot"
+	"github.com/alecthomas/kong"
 	"github.com/gorilla/websocket"
+	_ "github.com/mattn/go-sqlite3"
 )
 
 func main() {
-	http.Handle("/", http.FileServer(http.Dir("static")))
-	http.HandleFunc("/ws", socketHandler)
+	cfg := &foxtrot.Config{}
+	kong.Parse(cfg, kong.Description("Foxtrot Server"))
 
-	port := os.Getenv("PORT")
-	if port == "" {
-		port = "8080"
+	mux := http.NewServeMux()
+	mux.Handle("/", http.FileServer(http.Dir("static")))
+	mux.HandleFunc("/ws", socketHandler)
```

Comment:
So now dispatching to `socketHandler` in the handler function, you don't need this in the mux any more.

Reply:
yes. of course. i had it deleted in my first attempt.... done.